Submitted By: Joe Ciccone <jciccone@linuxfromscratch.org>
Date: 2007-03-10
Initial Package Version: 5.8.8
Upstream Status: From Upstream
Origin: http://www.nntp.perl.org/group/perl.perl5.changes/2006/07/msg16428.html
Description: Account for a change in gcc-4.2.0.

diff -Naur perl-5.8.8.orig/makedepend.SH perl-5.8.8/makedepend.SH
--- perl-5.8.8.orig/makedepend.SH	2007-03-10 08:55:07.000000000 -0500
+++ perl-5.8.8/makedepend.SH	2007-03-10 08:55:20.000000000 -0500
@@ -167,6 +167,7 @@
             -e '/^#.*<builtin>/d' \
             -e '/^#.*<built-in>/d' \
             -e '/^#.*<command line>/d' \
+            -e '/^#.*<command-line>/d' \
 	    -e '/^#.*"-"/d' \
 	    -e '/: file path prefix .* never used$/d' \
 	    -e 's#\.[0-9][0-9]*\.c#'"$file.c#" \
